DESCRIPTION

```diff
@@ -23,7 +23,12 @@ Authors@R:
     person(given = "Hannah",
            family = "Choi",
            role = "ctb",
-           email = "hannah.choi1@lshtm.ac.uk"))
+           email = "hannah.choi1@lshtm.ac.uk"),
+    person(given = "Carl",
+           family = "Pearson",
+           role = "ctb",
```

Comment:
This is enough for `aut` in my view now.

Reply:
still expecting to be reimbursed in :beers: